doc/source/whatsnew/v0.21.0.txt

```diff
@@ -40,7 +40,8 @@ Other Enhancements
 - :func:`DataFrame.clip()` and :func:`Series.clip()` have gained an ``inplace`` argument. (:issue:`15388`)
 - :func:`crosstab` has gained a ``margins_name`` parameter to define the name of the row / column that will contain the totals when ``margins=True``. (:issue:`15972`)
 - :func:`Dataframe.select_dtypes` now accepts scalar values for include/exclude as well as list-like. (:issue:`16855`)
-
+- :func:`read_csv` `read_html` `read_json` `read_html`  now accept auth in url //<user>:<password>@<host>:<port>/<url-path>, or ``auth`` tuple (username, password) parameter 
```

Comment:
@skynss : Sure thing.  **I would refrain from doing the detailed section right now**, as I would like to see what @jreback thinks of that suggestion.  The section would go right below the "Other Enhancements" title.  In that section, you would provide examples of how to use your newly added functionality.